app/views/authentications/failure.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Error</h1>
+
+<p>There was a problem logging you into the application. Please <%= link_to "try again", "/auth/#{params[:strategy]}" %>.</p>
```

Comment:
Could this copy be moved to locales? 

Is the page styled correctly? Doesn't the heading level need css class to style it? I was just thinking that we should use `govuk_link_to` (https://govuk-components.netlify.app/helpers/link/) instead of `link_to`

Reply:
The page has no styling currently (either in main or in this branch), I thought for quickness I'd leave the page pretty much as is.

Reply:
Ideally we'd be using the proper error layout and translations and all that, but I think that might need to involve changes to the routes and controllers?

Reply:
Yeah that makes perfect sense. I just didn't have context of what the existing page looked like. 

It might be worth adding to the PR description that its unstyled or screenshot for future devs looking at this PR.

Reply:
I've added screenshots to the description